src/python/pants/backend/jvm/subsystems/jvm_platform.py

```diff
@@ -226,17 +256,19 @@ class JvmPlatformSettings:
   class IllegalSourceTargetCombination(TaskError):
     """Illegal pair of -source and -target flags to compile java."""
 
-  def __init__(self, source_level, target_level, args, name=None, by_default=False):
+  def __init__(self, source_level, target_level, args, jvm_options, name=None, by_default=False):
     """
     :param source_level: Revision object or string for the java source level.
     :param target_level: Revision object or string for the java target level.
     :param list args: Additional arguments to pass to the java compiler.
+    :param list jvm_options: Additional jvm options specific to this JVM platform.
     :param str name: name to identify this platform.
     :param by_default: True if this value was inferred by omission of a specific platform setting.
     """
     self.source_level = JvmPlatform.parse_java_version(source_level)
     self.target_level = JvmPlatform.parse_java_version(target_level)
     self.args = tuple(args or ())
```

Comment:
Why don't we parse `self.args` and `self.jvm_options` the same way? I think it might be plausible that some subsystem might try to feed non-shlexed args from a pants list option into the `args` parameter as well.

Reply:
I originally updated args to be parsed with shlex, but wanted to narrow the effects of this commit. I could bring that back in.

Reply:
Thanks!!